UKHomeOffice/cerberus-service
# COP- 11203: Add test for Task information is not showing the other passenger details

## Description
Add test for Task information is not showing the other passenger details
https://support.cop.homeoffice.gov.uk/browse/COP-11203
## To Test
npm run cypress:test:local -- --spec cypress/integration/cerberus/airpax-task-details.spec.js

Should verify Co-Passenger details from target information sheet on task details page
## Developer Checklist

\* Required
